packages/developer-portal/src/components/pages/app-detail-preview/client/contact-developer-modal.tsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react'
```

Comment:
We can delete this component now I think

Reply:
I reused this component, just remove the modal the button to open it